docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -116,6 +117,7 @@ Table 1: Recommended compiler options that enable strictly compile-time checks.
 | [`-Wformat=2`](#-Wformat=2)                                                   | GCC 2.95.3<br/>Clang 4.0 | Enable additional format function warnings                                          |
 | [`-Wconversion`](#-Wconversion)<br/>[`-Wsign-conversion`](#-Wsign-conversion) | GCC 2.95.3<br/>Clang 4.0 | Enable implicit conversion warnings                                                 |
 | [`-Wtrampolines`](#-Wtrampolines)                                             |         GCC 4.3          | Enable warnings about trampolines that require executable stacks                    |
+| [`-Wimplicit-fallthrough`](#-Wimplicit-fallthrough)                           |         GCC<br>Clang   | Warn when a switch case falls through                                                 |
```

Comment:
GCC should support -Wimplicit-fallthrough since 7.1.0 [1]. I couldn't find a reference for Clang yet.

```suggestion
| [`-Wimplicit-fallthrough`](#-Wimplicit-fallthrough)                           |         GCC 7.1.0<br>Clang   | Warn when a switch case falls through                                                 |
```

[1] https://gcc.gnu.org/gcc-7/changes.html

Reply:
Clang appeared to have it by 2012: https://discourse.llvm.org/t/should-wimplicit-fallthrough-require-c-11/25199/13
... that doesn't answer the question, but perhaps it will help find the answer.